src/rendering/react/components/ProjectBackup/ProjectBackupDetails.tsx

```diff
@@ -0,0 +1,55 @@
+import React, { FC } from "react";
+import { MittwaldAPIV2 } from "@mittwald/api-client";
+import BackupProjectBackup = MittwaldAPIV2.Components.Schemas.BackupProjectBackup;
+import { SingleResult, SingleResultTable } from "../SingleResult.js";
+import { Text } from "ink";
+import { Value } from "../Value.js";
+import { useProjectBackupSchedule } from "../../../../lib/projectbackup/hooks.js";
+import { FormattedDate } from "../FormattedDate.js";
+import { ProjectBackupStatus } from "./ProjectBackupStatus.js";
+
+export const ProjectBackupDetails: FC<{
+  projectBackup: BackupProjectBackup;
+}> = ({ projectBackup }) => {
+  const schedule = projectBackup.parentId
+    ? useProjectBackupSchedule(projectBackup.parentId)
+    : undefined;
+
+  return (
+    <SingleResult
```

Comment:
should the project name be displayed in the details?

Reply:
Good idea. Fixed in 2dfb1b2